scoring/src/main/scala/ai/h2o/sparkling/ml/models/H2OMOJOPredictionMultinomial.scala

```diff
@@ -22,17 +22,26 @@ import org.apache.spark.sql.catalyst.expressions.GenericRowWithSchema
 import org.apache.spark.sql.expressions.UserDefinedFunction
 import org.apache.spark.sql.functions.col
 import ai.h2o.sparkling.sql.functions.udf
+import hex.genmodel.easy.EasyPredictModelWrapper
 import org.apache.spark.sql.types._
 import org.apache.spark.sql.{Column, Row}
 
 import scala.collection.mutable
 
 trait H2OMOJOPredictionMultinomial extends PredictionWithStageProbabilities {
   self: H2OAlgorithmMOJOModel =>
-  def getMultinomialPredictionUDF(): UserDefinedFunction = {
-    val schema = getMultinomialPredictionSchema()
+
+  private val predictionColType = StringType
+  private val predictionColNullable = true
+
+  def getMultinomialPredictionUDF(
+      schema: StructType,
+      modelUID: String,
+      mojoFileName: String,
+      configInitializers: Seq[(EasyPredictModelWrapper.Config) => EasyPredictModelWrapper.Config])
```

Comment:
What about introducing a type alias for: (EasyPredictModelWrapper.Config) => EasyPredictModelWrapper.Config

Reply:
Thanks for the suggestion, will do!